modules/nf-core/limma/differential/tests/main.nf.test

```diff
@@ -0,0 +1,284 @@
+nextflow_process {
+
+    name "Test Process LIMMA_DIFFERENTIAL"
+    script "../main.nf"
+    process "LIMMA_DIFFERENTIAL"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "limma"
+    tag "limma/differential"
+    tag "affy"
+    tag "affy/justrma"
+    tag "untar"
+
+
+
+    test("test_limma_differential") {
+
+        config "./nextflow.config"
+
+        setup {
+            run("UNTAR") {
+                script "../../../untar/main.nf"
+                process {
+                    """
+                    input[0] = [
+                        [id: 'test'],
+                        file(params.test_data['homo_sapiens']['genome']['affy_array_celfiles_tar'], checkIfExists: true)
```

Comment:
You need to convert these to the newer syntax you use below for consistency, using `params.modules_testdata_base_path`. You can find the map here: https://github.com/nf-core/modules/blob/master/tests/config/test_data.config

Reply:
Done :)